starters/vue3-xstate-css/src/machines/greetMachine.ts

```diff
@@ -1,5 +1,5 @@
 import { createMachine, assign } from 'xstate';
-import { fetchGreeting } from '@/utils/fetchGreeting';
+import { fetchGreeting } from '../utils/fetchGreeting';
```

Comment:
What was the reason for this change?
Was it for consistency with the other imports?

Reply:
I was testing out the build for storybook and it wasn't able to find the packages when I used the "@" sign. So I opted to use the relative reference instead of Vue's shortcut version. lol